hack/make/validate-vet

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+IFS=$'\n'
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^vendor/' || true) )
+unset IFS
+
+badFiles=()
```

Comment:
Removed badFiles, but we do need the other stuff I think. Also test failed because of an issue with jenkins